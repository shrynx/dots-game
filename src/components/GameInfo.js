// @flow
import styled from 'styled-components';

const GameInfo = styled.div`
  display: flex;
  background-color: #fff;
  height: 25vh;
`;

export default GameInfo;
