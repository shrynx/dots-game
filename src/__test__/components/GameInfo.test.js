import React from 'react';
import { render } from 'react-testing-library';
import GameInfo from '../../components/GameInfo';

it('GameInfo component should render correctly', () => {
  const { container } = render(<GameInfo />);
  expect(container).toMatchSnapshot();
});
