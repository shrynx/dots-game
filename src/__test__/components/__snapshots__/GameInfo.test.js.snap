// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameInfo component should render correctly 1`] = `
<div>
  <div
    class="sc-bdVaJa dCEpXi"
  />
</div>
`;
